release.yaml

```
@@ -82,6 +84,27 @@ promoteToPublic:
           fi
   finalize:
     steps:
-      - name: "Register on release registry"
+      - name: "Promote on release registry"
         cmd: |
           echo "pretending to call release registry api"
+
+      # tag is usually in the format `5.3.2`
+      # while version is usually the tag prepended with a v, `v5.3.2`
+      - name: "Slack notification (#announce-engineering)"
+        cmd: |
+          echo "Posting slack notification for release"
+          changelog_version="${tag//./}"
```

```suggestion
          tag="{{tag}}"
          changelog_version="${tag}//./}"
```

considering this is `changelog_version` should this not be `{{version}}` instead?

---

The URL format for the changelog has the period stripped off.

```
https://sourcegraph.com/docs/CHANGELOG#530
```

The version here is `530` instead of `5.3.0`. `{{version}}` is usually in the format `v5.3.0` which doesn't apply here.

---

I just noticed a mistake in my suggested fix